### DISCUSSION TITLE
Getting a preview for search results

### DISCUSSION THREAD

**Discussion by Fiday** [nextapps-de/flexsearch#228] [Q&A]
Hi, 
I'm trying to create a preview for the search results on my gitlab pages site built with Hugo. I'm pretty new to flexsearch and don't even know where the created Index file is saved/can be found (so testing is kinda hard)
Does anybody have some advice/examples of how it can be done? 

Thanks in advance
Fiday

**Comment by ts-thomas** [comment]
The index is in-memory. There is a method to export and import indexes, but persistent indexes are currently not supported.c

**Comment by schalkneethling** [comment]
@Fiday as @ts-thomas mentioned, the index is an `in-memory` index. The methods referred to are `export` and `import`. You can read more about using those here https://github.com/nextapps-de/flexsearch#export--import. I hope that helps.
